v2/tools/generator/internal/codegen/pipeline/report_resource_versions.go

```diff
@@ -183,6 +186,20 @@ func (report *ResourceVersionsReport) createTable(
 		return astmodel.ComparePathAndVersion(right.PackageReference.PackagePath(), left.PackageReference.PackagePath())
 	})
 
+	samplesMap := make(map[string]string)
+	err := filepath.WalkDir(report.samplesPath, func(filePath string, d fs.DirEntry, err error) error {
+		if !d.IsDir() && filepath.Base(filepath.Dir(filePath)) != "refs" {
+			sampleLink := filepath.Join(report.rootUrl, filePath)
```

Comment:
This will only give the right results if run on a Linux/MacOS system - use [`filepath.ToSlash`](https://pkg.go.dev/path/filepath#ToSlash) to convert the path so that this works on Windows too.